tests/files.output

```diff
@@ -0,0 +1,5 @@
+(* FreeSpec
```

Comment:
Weird file.

Reply:
This is the expected output of the `tests/files.v` test.